package.json

```diff
@@ -9,6 +9,6 @@
     "download-webviewer": "npx @pdftron/webviewer-downloader --webviewerLocation webapps/webviewer/lib"
   },
   "devDependencies": {
-    "@pdftron/webviewer-downloader": "^1.3.0"
```

Comment:
The `postinstall` script probably needs to be updated here too.

Reply:
done